### DISCUSSION TITLE
Welcome to The Chess Centre discussions!

### DISCUSSION THREAD

**Discussion by matt-d-webb** [chess-centre/platform#69] [General]
## 👋  Welcome!

  We’re using Discussions as a place to connect with members of our chess community, from our online apps perspective. We hope that you:
  * Ask questions you’re wondering about regarding our online platform.
  * Share ideas.
  * Engage with other community members on providing a rich, interactive digital experience.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what brought you here. ♟️

**Comment by akashsiddamshetty** [comment]
hello! My self akash siddamshetty 
i am looking to contribute into the open source projects

**Reply by matt-d-webb** [reply]
Great to hear. As mentioned in my email, I am just testing the contributor model now so will let you know if any additional tweaks to the set up are required. :-)

**Reply by matt-d-webb** [reply]
@akashsiddamshetty  Okay, so there is definitely some permissions I need to tweak:

![image](https://user-images.githubusercontent.com/36933715/147880407-c8b26803-57ed-4cdd-aaca-c31c5fa5e963.png)

Hang fire.
